siac/hostcmd.go

```diff
@@ -55,8 +56,7 @@ func hostconfigcmd(param, value string) {
 		// convert price to hastings/byte/block
 		p, ok := new(big.Rat).SetString(value)
 		if !ok {
-			fmt.Println("could not parse price")
-			return
+			dieNoError("Could not parse price")
```

Comment:
That is much more elegant but it would require the callers to add "\nError:" in order to be equivalent to the current output.

What do you think about the following?
```go
func die(context string, err error) {
 	fmt.Fprintln(os.Stderr, context)
 	if (err != nil) {
		fmt.Fprintln(os.Stderr, "Error:", err)
	}
 	os.Exit(exitCodeGeneral)
 }
```

I'm in favor of keeping the current output style as it more closely matches cobra's output when you run an invalid command e.g. `siac foo`